cmd/containerd/command/service_windows.go

```diff
@@ -79,6 +84,10 @@ func serviceFlags() []cli.Flag {
 			Name:  "log-file",
 			Usage: "Path to the containerd log file",
 		},
+		cli.StringFlag{
+			Name:  "priorityclass",
```

Comment:
Can we change the flag name to be `priority-class`, for consistency with other flags?